### PR TITLE
Mekko for district heating HT, MT, LT

### PR DESCRIPTION
**Pull request by kndehaan**
This PR adds the following 3 Mekko charts:
- District heating HT
- District heating MT
- District heating LT

Goes with https://github.com/quintel/etmodel/pull/4380

### CODE REVIEW

**Review by mabijkerk — COMMENTED** [review]
I'm not entirely sure why the heat output of the backup burners is in the folder "heat_production_per_source". It seems as those that folder should be "_per_carrier" instead. The backup burner might then be moved to "output_of_steam_hot_water_per_technology", which we might rename to "heat_production_per_technology" for consistency?

File: graphs/energy/edges/other/other_final_demand_for_heat_ht_steam_hot_water-other_heater_district_heating_ht_steam_hot_water@steam_hot_water.ad
Comment (mabijkerk) [inline graphs/energy/edges/other/other_final_demand_for_heat_ht_steam_hot_water-other_heater_district_heating_ht_steam_hot_water@steam_hot_water.ad:1]: Nice catch!

**Review by mabijkerk — COMMENTED** [review]
Would it be useful to more clearly label supply and demand in the mekko queries? For example, by starting all the supply queries with "supply_of" and all the demand queries with "demand_for".

File: gqueries/mechanical_turk/mekko/turk_industrial_heat_mekko_supply.gql
Comment (mabijkerk) [inline gqueries/mechanical_turk/mekko/turk_industrial_heat_mekko_supply.gql:1]: We did not add a mekko for industry steam network in this PR right?

File: gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_supply.gql
Comment (mabijkerk) [inline gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_supply.gql:1]: What value does this query add, compared to the query `turk_mekko_of_collective_heat_ht_network_total_demand`? Wouldn't a `FILTER` on G(heat_production) and steam hot water output of `heat_network_ht.type = producer` be more useful?

File: gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_demand.gql
Comment (mabijkerk) [inline gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_demand.gql:1]: Creative approach!

File: gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_supply.gql
Comment (kndehaan) [reply gqueries/mechanical_turk/mekko/turk_mekko_of_collective_heat_ht_network_total_supply.gql:1]: Good point, didn't think of doing it this way but looks more logical. I now changed this for HT, MT, LT.

**Comment by kndehaan** [comment]
> Follow-up: the difference disappears when I remove the installed capacity for the power-to-heat heat pump. Could you have a look to see if this is indeed the culprit in our balancing issues @kndehaan?

It definitely looks like there are some issues with balancing of p2h heat pump.Your described scenario without changing installed capacity of p2h heat pumps gives the following results of respectively total `output_of_steam_hot_water` of the node and the sum of the `steam_hot_water_output_curve`:

```
[
  7.2355604597210865,
  7.232994379721087
]
```

**Comment by kndehaan** [comment]
> Would it be useful to more clearly label supply and demand in the mekko queries? For example, by starting all the supply queries with "supply_of" and all the demand queries with "demand_for".

Good suggestion, however I feel that this is now too much if an exhaustive effort to adjust this in retrospect. Will take this up with the new mekkos to come.

**Comment by mabijkerk** [comment]
See https://github.com/quintel/etsource/issues/3165 for balancing issue.

**Review by mabijkerk — APPROVED** [review]

**Pull request merged**